src/main/java/edu/njit/jerse/ashe/services/SpeciminTool.java

```diff
@@ -126,7 +161,7 @@ private static List<String> prepareCommands(String speciminPath, String argsWith
     /**
      * Logs the commands being executed.
      *
-     * @param commands List of commands to be logged.
+     * @param commands {@link List} of commands to be logged
```

Comment:
This `{@link ...}` is not needed, because in the Javadoc, the return type will be linked.  The same comment goes for anywhere else that there is a `{@link ...}` for a type that is in a signature.